### ISSUE TITLE
Code blocks in the document cannot be copied

### ISSUE THREAD

**Issue by tzSharing** [vlang/docs#11]
Code blocks in the document cannot be copied. We want to support copy operations.

**Comment by spytheman** [comment]
Yes, it is very annoying.

**Comment by ctkjose** [comment]
@spytheman Had you had a chance to look at this.

**Comment by ctkjose** [comment]
I noticed that this code is using a very dated version of CodeMirror.  I made a separate fork and got a newer Codemirror 5 working. I fixed a couple of js glitches and did some cleanup. Since we already have full fledge playground at https://play.vlang.io I decided to remove the playground functionality and instead code snippets now take you to the new playground, also attempting to fix the minified version of vlang-playground.js is just not worth it.

The clipboard, search and other few errors were fixed.

In my version I cleaned the js and css gave and re-focus the generator on its roles as a documentation navigator. 

My refactoring is at this new [repo](https://github.com/ctkjose/v-docs-generator).
